src/gui/src/displayControls.cpp

```diff
@@ -333,6 +333,9 @@ DisplayControls::DisplayControls(QWidget* parent)
     pin_markers_font_ = QFontDialog::getFont(nullptr, pin_markers_font_, this, "Pin marker font");
   });
 
+  // Access Points
+  makeParentItem(access_points_, "Access Points", model_, Qt::Checked);
```

Comment:
I would put this in Misc as a leafItem

You are missing the code to read/write this in the settings.